subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -401,6 +414,13 @@ static int isr_rx_pdu(struct lll_scan_aux *lll, uint8_t devmatch_ok,
 		bt_addr_t *lrpa;
 #endif /* CONFIG_BT_CTLR_PRIVACY */
 
+		/* FIXME: we need lll_aux for aux_connect_req handling, need to
+		 *        fix somehow
+		 */
+		if (!lll_aux) {
```

Comment:
Correct me if I'm wrong. This will return -ENOBUFS if received ADV_EXT_IND PDU with pointer `aux_ptr` to auxiliary PDU with offset short enough to do not go through ULL (schedule next PDU from lll_scan).

Reply:
That is just temporary until I update it to handle aux_adv_ind scheduled from LLL on connectable, so this will change in final version.